impeller/renderer/backend/gles/proc_table_gles.h

```diff
@@ -85,7 +92,6 @@ struct GLProc {
   constexpr bool IsAvailable() const { return function != nullptr; }
 
   void Reset() {
-    name = nullptr;
```

Comment:
Can we shift name to a string_view?  A raw pointer is unclear what the memory rules are.

Reply:
I actually removed resetting the name, because it means we lose context on error messages.

Did you still want it changed?

Reply:
Up to you, string_view's are still probably preferable because they explicitly say they aren't taking ownership, while with c strings it's ambiguous.

Reply:
Ack. I am going to file a follow-up issue to keep the scope small:

https://github.com/flutter/flutter/issues/135922